src/main/java/frc/robot/subsystems/simulated/ArmSimulation.java

```diff
@@ -25,7 +26,7 @@ public class ArmSimulation {
     public static final double ARM_BASE_LENGTH_M = Units.inchesToMeters(28);
     public static final double ARM_GEAR_REDUCTION = 89;
     public static final double EXTENSION_MAX_SPEED_M_S = 0.25;
-    public static final double GRIPPER_MAX_SPEED_M_S = 90.0;
+    public static final double GRIPPER_MAX_SPEED_M_S = 9.0;
```

Comment:
This is why the gripper in the simulator was twitching wildly. Tuned its max speed down so it doesn't teleport back and forth

(This is a side-effect of simulating object motion as opposed to real-world motion: because things in the sim don't actually "move", they make little micro-teleports from position to position 50 times a second, if you "move" them too fast they can blip right past a constraint that would stop them in the real world.

Halo 2 had the problem that you could throw grenades through walls if you got the angle just right; same deal).